src/test/java/frc/robot/subsystems/gpm/ShooterTest.java

```diff
@@ -48,40 +51,41 @@ public void conversionTest() {
             0.0001);
     }
 
+    @Disabled
```

Comment:
Nice job with @Disabled. Good to remember, as when done with tests, can do it rather than commenting out everything.

Reply:
The other advantage of @Disabled is the test report says the unit test has been skipped. If it were commented out, then it would essentially disappear.